cmd/stack.go

```diff
@@ -215,3 +206,99 @@ func stringifyOutput(v interface{}) string {
 
 	return string(b)
 }
+
+type treeNode struct {
+	res      *resource.State
+	children []*treeNode
+}
+
+func renderNode(node *treeNode, padding, branch string, showURNs, showIDs bool, rows *[]cmdutil.TableRow) {
+	padBranch := ""
+	switch branch {
+	case "├─ ":
+		padBranch = "│  "
+	case "└─ ":
+		padBranch = "   "
```

Comment:
any chance this rendering coudl somehow be shared with the tree rendering in the progress display?

i would hope that given a simple interface over a tree+labels this wouldn't be too hard to generate a proper string-row just for the tree portion.

No biggie though.  just seems like it would be nice.

Reply:
The two systems seem so divergent otherwise that sharing this code seems unlikely (and perhaps unwise, depending on how stark the differences are). Maybe in the future.

Reply:
no worries :)